code/controllers/subsystem/pai.dm

```diff
@@ -31,6 +31,11 @@ SUBSYSTEM_DEF(pai)
 
 			candidates -= candidate
 			usr << browse(null, "window=findPai")
+		if (!candidates.len)
+			for(var/obj/item/paicard/p in pai_card_list)
+				if(!p.pai)
+					p.clearalert()
+
```

Comment:
nest this down another level

Reply:
still doesn't seem to be working by my testing